index.html

```diff
@@ -2833,136 +2833,105 @@ <h3>Consumption</h3>
-cannot be registered by the time the rest of DID Core is ready for W3C
-Proposed Recommendation. See also <a href="https://github.com/w3c/did-core/issues/208">
-Issue 208</a>.
+JSON-LD [[JSON-LD11]] is a JSON-based format used to serialize <a
+href="http://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. This
+section defines the <a>production</a> and <a>consumption</a> rules for the
+JSON-LD <a>representation</a>.
       </p>
 
       <p>
-The following application-specific modifications are made by this specification
-to the JSON-LD specification [[JSON-LD11]] to ease interoperability between
-JSON and JSON-LD implementations:
+The JSON-LD <a>representation</a> defines the following representation-specific
+entries:
       </p>
 
-      <ul>
-        <li>
-The <code>@id</code> and <code>@type</code> keywords are aliased to
-<code><a>id</a></code> and <code>type</code> respectively, enabling developers
-to use this specification as idiomatic JSON.
-        </li>
-        <li>
-Even though JSON-LD allows any IRI as node identifiers, <a>DID documents</a>
-are explicitly restricted to only describe <a>DIDs</a>. This means that the value of
-<code><a>id</a></code> that refers to the <a>DID subject</a> MUST be a valid
-<a>DID</a> and not any other kind of IRI.
-        </li>
-        <li>
-Data types, such as integers, dates, units of measure, and URLs, are
-automatically typed to provide stronger type guarantees for use cases that
-require them.
-        </li>
-      </ul>
+      <dl>
+        <dt><dfn>@context</dfn></dt>
+        <dd>
+The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a>
+is either a <a data-cite="INFRA#string">string</a> or a <a
+data-cite="INFRA#list">list</a> containing any combination of <a
+data-cite="INFRA#string">strings</a> and/or <a data-cite="INFRA#maps">ordered
+maps</a>.
+        </dd>
+      </dl>
 
       <section>
         <h3>Production</h3>
 
         <p>
-The <a>DID document</a> MUST be serialized according to the
-<a href="#production">production rules for JSON</a>, with one additional
-requirement: The serialized <a>DID document</a> MUST include the
-<code>@context</code> member.
+The <a>DID document</a> and any DID document data structures expressed by the <a href="#data-model">data model</a> MUST
+be serialized to the JSON-LD <a>representation</a> according to the JSON
+<a>representation</a> <a>production</a> rules as defined in <a
+href="#json"></a>.
         </p>
 
-        <dl>
-          <dt>@context</dt>
-          <dd>
-
-            <p>
-The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD
-specification</a> defines values that are valid for this member. It
-contains representation-specific syntax and therefore could be present as an
-entry in the <a
-href="#data-model">data model</a> to aid in lossless conversion. If the entry
-is present in the <a href="#data-model">data model</a>, it MUST be used during
```

Comment:
For the record, this language was added quite recently in https://github.com/w3c/did-core/pull/454 after much discussion, and it has been important for those of us who DON'T think `@context` is a REQUIRED entry in the abstract data model. Removing this part could therefore be problematic. But it looks like in https://github.com/w3c/did-core/issues/664 we're on a good path to make everyone happy!